### PR TITLE
add options to stop filtering by `PASS`

### PR DESCRIPTION
**Pull request by famosab**
## PR checklist
Related to #9 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/qbic-pipelines/vcftomaf/tree/master/.github/CONTRIBUTING.md)
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

We want to stop assuming that everybody just wants the `PASS` variants in the final MAF file. This is now an optional input :)

### CODE REVIEW

**Comment by famosab** [comment]
Also added the version bump to prepare the next release

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,21 +3,24 @@
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.1.0dev
+## v1.1.0 - 22-11-2024 - Amaranth Filly
 
 ### `Added`
 
-- [#11](https://github.com/qbic-pipelines/vcftomaf/pull/11) - Add metromap
+- [#11](https://github.com/qbic-pipelines/vcftomaf/pull/11) - Add metromap (@FriederikeHanssen)
+- [#18](https://github.com/qbic-pipelines/vcftomaf/pull/18) - Add options to stop filtering by `PASS` (@famosab)
```
Comment (d4straub) [inline CHANGELOG.md]:
```suggestion
- [#18](https://github.com/qbic-pipelines/vcftomaf/pull/18) - Add options to enable filtering by `PASS` via `--filter`, filtering was default in previous version (@famosab)
```
Hm I think its the other way around. The option allows to optionally switch on PASS filtering (whatever that is), in contrast the previous default was to have it switched on. So you changed the default, right? If yes, thats sort of noteworthy imho.

File: conf/modules.config [hunk 2]
```diff
@@ -19,7 +19,10 @@ process {
     ]
 
     withName: BCFTOOLS_VIEW {
-        ext.args = ' -f PASS --output-type z'
+        ext.args = { [ "",
```
Comment (d4straub) [inline conf/modules.config:22]: why an empty argument? Is that needed? Have never seen that

**Review by d4straub — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (famosab) [reply CHANGELOG.md]: yes you are right thats more descriptive

File: conf/modules.config (hunk 2 above)
Comment (famosab) [reply conf/modules.config:22]: probably not, tbh I copied that from below (VCF2MAF)

Comment (famosab) [reply conf/modules.config:22]: I would leave it in for now.

**Review by FriederikeHanssen — COMMENTED** [review]
Isn't filtering by PASS much more standard than skipping the filtering?

**Comment by famosab** [comment]
> Isn't filtering by PASS much more standard than skipping the filtering?

Depends, TCGA never filtered their results by PASS. Usability wise it probably does not matter. Decided to set it to not filter per standard because Leon ran into these issues.
But most people are probably only interested in PASS filtered variants?

**Comment by FriederikeHanssen** [comment]
> > Isn't filtering by PASS much more standard than skipping the filtering?
> 
> Depends, TCGA never filtered their results by PASS. Usability wise it probably does not matter. Decided to set it to not filter per standard because Leon ran into these issues. But most people are probably only interested in PASS filtered variants?

In my experience you just get a ton of noise with the clinical data if you don't filter. This has at least tripped me up many times.

**Pull request merged**